src/components/Onboarding/kit/ChartComponents.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Card, GU, textStyle, useTheme } from '@1hive/1hive-ui'
+
+export const ChartBase = ({ children, height, title, width }) => (
+  <div>
+    {title && (
+      <div
+        css={`
+          margin-bottom: ${1 * GU}px;
```

Comment:
```suggestion
          text-align: center;
          margin-bottom: ${1 * GU}px;
          ${textStyle('body1')};
```
It can be nice to center the titles of the graphs and style then a bit.